ui/com/composer/index.jsx

```diff
@@ -55,20 +56,19 @@ export default class Composer extends React.Component {
     if (this.props.thread) {
       // thread categorization
       this.isThreadPublic = isThreadPublic(this.props.thread)
-      this.threadChannel = (this.props.thread.value.content.channel || '').trim()
+        this.threadChannel = (this.props.thread.value.content.channel || '').trim()
```

Comment:
whoops! crap, I thought I got all of these. JSX makes a lot of linters and auto-indenters lose their minds ;)